packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -647,6 +651,309 @@ describeWithDOM('mount', () => {
 </Foo>`);
     });
   });
+  describeIf(is('>= 16.8'), 'hooks', () => {
+    describe('useState', () => {
+
+      /*
+        Test Component
+      */
+      function FooCounter({ initialCount = 0 }){
+        const [count, setCount] = useState(+initialCount);
+
+        return (
+          <Fragment>
+              <button className="increment" type="button" onClick={() => setCount( count + 1)}>-</button>
+                <span className="counter">
+                  {count}
+                </span>
+              <button className="decrement" type="button" onClick={() => setCount( count - 1)}>+</button>
+          </Fragment>
+        );
+      };
+      
+      // Test data
+      let wrapper;
+      const initialCount = 5;
+
+      before(() => {
+        wrapper = mount(<FooCounter initialCount={initialCount}/>);
+      });
+
+      it('initial render', () => {
+        expect(wrapper.find('.counter').text()).to.equal(initialCount.toString());
```

Comment:
```suggestion
        expect(wrapper.find('.counter').text()).to.equal(String(initialCount));
```

Reply:
@ljharb just to confirm you are requesting to assign to variable and then assert, right ?

Reply:
suggested changes done dfdfd0546c210aff1bb560dc710e7b817332b583